tests/jenkins/tests/test_view_single_result.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
We used to add the licence header to all files, however now that we have the licence file in the root of the repository this is no longer necessary, so we decided to remove it to avoid the boilerplate. Could you remove the header from all files?